amimals/Fish.java

```diff
@@ -0,0 +1,29 @@
+package amimals;
+
+public class Fish extends Animals{
+    private String typeOfFish;
```

Comment:
Fish - name of class, field also can be just type. But if animal has its own type, then yes, it is necessary